jkube-kit/jkube-kit-spring-boot/src/main/java/org/eclipse/jkube/springboot/generator/SpringBootGenerator.java

```diff
@@ -84,7 +84,7 @@ public List<ImageConfiguration> customize(List<ImageConfiguration> configs, bool
 
     @Override
     protected Map<String, String> getEnv(boolean prePackagePhase) {
-        return nestedGenerator.getEnv(ppp -> super.getEnv(ppp), prePackagePhase);
+        return nestedGenerator.getEnv(super::getEnv, prePackagePhase);
```

Comment:
Apparently, the statement is incompatible with Java 8.
Let's just remove the warning and delay the refactoring to the future:
```suggestion
      //TODO: Java8 compatibility, remove warning. Remove annotation once baseline is set to Java11+
      //noinspection Convert2MethodRef
      return nestedGenerator.getEnv(ppp -> super.getEnv(ppp), prePackagePhase);
```